consensus/cryptarchia-engine/src/lib.rs

```diff
@@ -143,7 +143,7 @@ impl<Id> Cryptarchia<Id>
 where
     Id: Eq + std::hash::Hash + Copy,
 {
-    pub fn from_genesis(id: Id, config: Config) -> Self {
+    pub fn new(id: Id, config: Config) -> Self {
```

Comment:
Could there be a clearer name for this method?

Reply:
I'd argue it's quite accurate. It's really not doing anything fancy, nor anything attached to a special state (genesis), just getting an instance with a given `id`.

Reply:
Naming is fine I think.